### PR TITLE
Remove the multi-arch build pipeline

### PR DESCRIPTION
**Pull request by brunoapimentel**
This PR is a proposal to remove the multi-arch build pipeline from Github Actions, since it will become redundant with the move to the RHTAP pipelines.

Please note that RHTAP currently does not have the capability to build ppc64 images, which means that the support for this arch will be discontinued with this change. 

# Maintainers will complete the following section

- [x] Commit messages are descriptive enough
- n/a Code coverage from testing does not decrease and new code is covered
- n/a New code has type annotations
- [x] Docs updated (if applicable)
- [x] Docs links in the code are still valid (if docs were updated)

### CODE REVIEW

**Comment by brunoapimentel** [comment]
@mayurwaghmode We've been trying to reach you by internal means without success. Please let us know if there's any objections to this PR.

**Comment by mayurwaghmode** [comment]
> Should be merged after #101.
> 
> This PR is a proposal to remove the multi-arch build pipeline from Github Actions, since it will become redundant with the move to the RHTAP pipelines.
> 
> Please note that RHTAP currently does not have the capability to build ppc64 images, which means that the support for this arch will be discontinued with this change.
> 
> # Maintainers will complete the following section
> * [ ]  Commit messages are descriptive enough
> * [ ]  Code coverage from testing does not decrease and new code is covered
> * [ ]  New code has type annotations
> * [ ]  Docs updated (if applicable)
> * [ ]  Docs links in the code are still valid (if docs were updated)



> Should be merged after #101.
> 
> This PR is a proposal to remove the multi-arch build pipeline from Github Actions, since it will become redundant with the move to the RHTAP pipelines.
> 
> Please note that RHTAP currently does not have the capability to build ppc64 images, which means that the support for this arch will be discontinued with this change.
> 
> # Maintainers will complete the following section
> * [ ]  Commit messages are descriptive enough
> * [ ]  Code coverage from testing does not decrease and new code is covered
> * [ ]  New code has type annotations
> * [ ]  Docs updated (if applicable)
> * [ ]  Docs links in the code are still valid (if docs were updated)

@Raja-0940 FYI

**Comment by chmeliik** [comment]
We should also update the pullspecs from quay.io/containerbuildsystem to quay.io/redhat-appstudio in the README

File: README.md [hunk 1]
```diff
@@ -314,8 +314,8 @@ still in early development phase.
 
 [cachi2-coveralls]: https://coveralls.io/github/containerbuildsystem/cachi2?branch=main
 [cachi2-coveralls-badge]: https://coveralls.io/repos/github/containerbuildsystem/cachi2/badge.svg?branch=main
-[cachi2-container]: https://quay.io/repository/containerbuildsystem/cachi2
-[cachi2-container-status]: https://quay.io/repository/containerbuildsystem/cachi2/status
+[cachi2-container]: https://quay.io/repository/redhat-appstudio/cachi2
+[cachi2-container-status]: https://quay.io/repository/redhat-appstudio/cachi2/status
```
Comment (chmeliik) [inline README.md:318]: Still some occurrences left

```shell
$ grep -R 'quay.io/.*containerbuildsystem' --include='*.md'
README.md:quay.io/containerbuildsystem/cachi2:latest
README.md:alias cachi2='podman run --rm -ti -v "$PWD:$PWD:z" -w "$PWD" quay.io/containerbuildsystem/cachi2:latest'
README.md:CACHI2_IMAGE=quay.io/containerbuildsystem/cachi2:{tag} tox -e integration

```

**Review by chmeliik — APPROVED** [review]
LGTM :+1:

**Review by lkolacek — APPROVED** [review]
Looks great :+1:

**Pull request merged**